libraries/core_libs/consensus/src/pbft/pbft_manager.cpp

```diff
@@ -1607,6 +1607,7 @@ std::pair<vec_blk_t, bool> PbftManager::compareBlocksAndRewardVotes_(std::shared
 
   auto dag_order_it = anchor_dag_block_order_cache.find(anchor_hash);
   if (dag_order_it != anchor_dag_block_order_cache.end()) {
+    period_data_.pbft_blk = std::move(pbft_block);
```

Comment:
no need to do anything also with period_data_.transactions & period_data_.dag_blocks?

Reply:
Refactored PbftManager::compareBlocksAndRewardVotes_, period_data_ is no longer set here